projects/plugins/crm/admin/settings/field-sorts.page.php

```diff
@@ -168,11 +168,7 @@
 								<?php
 								// } output fields
 
-								// } Weird this doesn't work for mike:
-								// global $$fieldType['obj'];
-								$fieldTypeObjVarName = $fieldType['obj'];
-								global ${$fieldTypeObjVarName};// $$fieldTypeObjVarName; #} compat php7
-								$fieldTypesArray = ${$fieldTypeObjVarName};
+								$fieldTypesArray = isset( $GLOBALS[ $fieldType['obj'] ] ) ? $GLOBALS[ $fieldType['obj'] ] : null;
```

Comment:
The default value should be an empty array. If `$fieldTypesArray` is `null`, then the condition below (line 180) that uses `count` could give us a Warning notice.

Reply:
This code does not represent a change in the existing behavior. `global $foo`, if `$foo` is not set, results in `$foo` having a null value.

Reply:
Yes, that is, it doesn't change the current one. We will address it in another issue to add this check added.